assets/sass/patterns/atoms/see-more-link.scss

```diff
@@ -10,6 +10,9 @@
     content: "\00a0\00bb";
   }
 
+  .content-aside__related-item & {
+    @include margin(24, "top");
```

Comment:
Can we not add this to the bottom of the  .listing-list__item class  rather than putting the margin on the top?

We try and put the margin and padding on the bottom of items unless there is no other way

Reply:
I added the margin on the top because "see-more-link" is optional and it will increase the distance after the component. It is not a problem to decrease component bottom space and add it on "listing-list__item" class.

Reply:
Done.